guides/common/assembly_managing-dns-on-smart-proxies.adoc

```diff
@@ -1,5 +1,9 @@
 include::modules/con_managing-dns-using-smart-proxy.adoc[]
 
+include::modules/proc_configuring-dns-nsupdate.adoc[leveloffset=+1]
+
+include::modules/proc_installing-the-dns-infoblox-module.adoc[leveloffset=+1]
```

Comment:
If you include info only from this module, it's missing the Infoblox limitations and prerequisites info. Is it possible that some of that information only applies to combining DNS and DHCP Infoblox modules? Not sure how to handle this content-wise.

Reply:
I think you're right that some of them should be included. However, the chapter really is written to reflect on both DHCP and DNS. I wonder if it makes sense to move the prerequisites into the specific chapters. But I also see the CA certificate installation step is missing.

How about we exclude Infoblox for now and handle it as a separate follow up?

Reply:
Sure, that's an option.